MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+
+For greater decentralization, we propose the following:
+- Provide information and onboarding materials to the community to empower members and give them more agency in the decision-making processes.
+- Implement the system on-chain (eventually, once tested) so that processes are automated, eliminating dependency on the Mina Foundation to manage these processes manually.
+
+
+## Specification
+
+The following is a description of the proposed changes to the MIP process:
+
+### New MIP Categories
+
+Currently, the MIP process is narrowly focused on technical types of protocol changes and decisions. To support more flexible decision making, we propose to broaden the MIP process to include the following three categories. 
+
+**Engineering MIPs** would continue to focus on Mina's core protocol technical foundations and what feature upgrades the protocol should undergo. For example:
+- The network protocol
+- The consensus mechanism
+- Block validity rules
+- Proof system
+
+**Economics MIPs** would consider Mina's financial parameters, the levels and sources of funding to support development, including how a decentralized treasury should be configured and managed. For example:
+- Mina tokenomics
+- Transaction fees
+- Treasury management
+
+**Governance MIPs** would consider the rules and processes for making changes to the protocol and funding its development, and how they can be improved over time. For example:
+- MIP process
+- MIP roles and processes
+- On-Chain Voting
+
+These categories will need to be clearly defined, specifying their parameters and what kinds of major changes could be made.
+
+### New Roles and Responsibilities
+
+To encourage greater community participation, we propose to create the following roles while ensuring that ultimate decision-making authority remains with the community through on-chain voting:
+- MIP Author
+- MIP Facilitator
+- MIP Reviewer
+- Dev Team
+
+#### Clarifying Responsibilities of Existing Roles: 
+
+**MIP Author:** any community member can present a proposal and start the MIP process.
+
+Responsibilities: 
+- Articulate the proposed changes clearly and concisely.
+- Explain the expected effects of the proposed changes.
+- Encouraging community participation and engaging in discussions around the proposal.
+- Actively seek feedback from other community members.
+- Check alignment with the community.
+
+Functions:
+- Draft, edit and refine the MIP using the provided template.
+- Follow the instructions provided in the GitHub/MIPs documentation.
+- Collect and incorporate feedback from the community.
+- Document alternative opinions, the options considered and other discussed topics.
+- Champion the MIP through each stage of the process.
+- Designate, coordinate and oversee a Dev Team for the MIP implementation.
```

Comment:
I'm wondering if requiring the MIP author to also coordinate the delivery is too constraining. Authoring an MIP and handling the implementation require different skillsets, so having one person do both might not always be best.

Maybe we should allow for the MIP author to be a team, instead of an individual, natural person? That might make it easier to get all the skills required.

Reply:
I completely agree, and this is already addressed in the proposed template. As it states: _Multiple authors can contribute to one MIP, but at least one person must be identified to submit and edit it._

That said, I understand how this might not be immediately clear. I’ll make it more explicit across all documents describing the MIP author role to ensure there’s no confusion. Thank you for bringing this up—it’s a valuable point!